README.md

```diff
@@ -442,3 +441,242 @@ receiver.addHandler(new ExampleReceiver() {
 });
 connection.call("interfaceName", "methodName", args, receiver);
 ```
+
+#### @Proxy
+
+`@Proxy` annotation allows you to auto-generate interface to the remote
+server that will look as if you are calling local java methods easily. To create
```

Comment:
`@Proxy` annotation allows you to auto-generate interface to the remote
 +server that will look as if you are calling local java methods easily
->
`@Proxy` annotation allows you to easily auto-generate interface to the remote
 +server that will look as if you are calling local java methods